roles/middleware/files/FWO.Middleware.Server/AutoDiscoverScheduler.cs

```diff
@@ -138,6 +141,8 @@ private async void AutoDiscover(object? _, ElapsedEventArgs __)
                     {
                         Log.WriteError("Autodiscovery", $"Ran into exception while auto-discovering management {superManagement.Name} (id: {superManagement.Id}) ", excMgm);
                         ActionItem actionException = new ActionItem();
+                        actionException.Number = 0;
+                        actionException.ActionType = ActionCode.DeleteManagement.ToString();
```

Comment:
Maybe we should define another ActionCode like "AutodiscoveryError" or "DoNothing" and handle it separately in AutoDiscovery.razor -> AnalyzeActions() (by displaying error message, maybe a more detailed instruction...)

Reply:
fixed